app/services/like_service.rb

```diff
@@ -5,11 +5,17 @@ def initialize(user=nil)
     @user = user
   end
 
-  def create(post_id)
+  def create_for_post(post_id)
     post = post_service.find!(post_id)
     user.like!(post)
   end
 
+  def create_for_comment(comment_id)
+    comment = comment_service.find_by_id!(comment_id)
```

Comment:
Rails/DynamicFindBy: Use `find_by!` instead of dynamic `find_by_id!`.

Reply:
Is it Ok to have "find_by!" and "find_by_guid"?